src/status_im/contexts/wallet/create_account/view.cljs

```diff
@@ -119,9 +120,8 @@
          :track-text          (i18n/label :t/slide-to-create-account)
          :customization-color @account-color
          :on-auth-success     (fn [entered-password]
-                                (prn entered-password)
                                 (rf/dispatch [:wallet/derive-address-and-add-account
-                                              {:sha3-pwd     entered-password
+                                              {:sha3-pwd     (security/safe-unmask-data entered-password)
```

Comment:
@J-Son89 I added the change to mask the pwd inside authorize's `on-auth-success` to avoid the password hash getting leaked e.g. as it would in line 122 `(prn entered-password)`, as it's still sensitive info.